modules/_labs/color-picker/react/stories/stories_VisualTesting.tsx

```diff
@@ -0,0 +1,44 @@
+/// <reference path="../../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {colors} from '@workday/canvas-kit-react-core';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {action} from '@storybook/addon-actions';
+import {ComponentStatesTable, withSnapshotsEnabled} from '../../../../../utils/storybook';
+import ColorPicker from '../lib/ColorPicker';
+
+export default withSnapshotsEnabled({
+  title: 'Testing|React/Labs/Color Picker',
+  component: ColorPicker,
+});
+
+export const ColorPickerStates = () => (
+  <StaticStates>
+    <ComponentStatesTable
+      rowProps={[
+        {label: 'Default', props: {}},
+        {label: 'with Hex Input', props: {showCustomHexInput: true}},
+        {
+          label: 'With Reset',
+          props: {
+            resetColor: colors.blueberry400,
+            resetLabel: 'Reset',
+            onColorReset: action('Reset Clicked'),
+          },
+        },
+        {
+          label: 'With Reset and Hex Input',
+          props: {
+            showCustomHexInput: true,
+            resetColor: colors.blueberry400,
+            resetLabel: 'Reset',
+            onColorReset: action('Reset Clicked'),
+          },
+        },
+      ]}
+      columnProps={[{label: 'Default', props: {}}]}
+    >
+      {props => <ColorPicker {...props} onColorChange={action('Color Changed')} />}
```

Comment:
@jtschult since the color picker animates in, you need to add `transformOrigin={null}` as a prop so that the snapshot get covered. Chromatic isn't the greatest when we have things that animate into frame.

Reply:
And then in the `ColorPicker` you can add the `transformOrigin` prop like this: https://github.com/Workday/canvas-kit/blob/master/modules/toast/react/lib/Toast.tsx#L41

https://github.com/Workday/canvas-kit/blob/master/modules/toast/react/lib/Toast.tsx#L92

Reply:
Thank you